lib/plugins/aws/package/compile/events/httpApi.js

```diff
@@ -274,15 +274,39 @@ Object.defineProperties(
       const userAuthorizers = userConfig.authorizers;
       const authorizers = (this.config.authorizers = new Map());
       if (userAuthorizers) {
+        if (typeof userAuthorizers !== 'object') {
```

Comment:
This type of checks should be implemented at schema level and I believe it's already in place: https://github.com/serverless/serverless/blob/master/lib/plugins/aws/provider.js#L720 Please correct me if I'm wrong

Reply:
If it exists it doesn't work. I only added more errors for problems that hit me with cryptic errors (undefined references for example)

Reply:
Could you share a specific reproducible example where that does not work? I've just checked on `2.19.0` with the following config:

```
service: my-service
configValidationMode: error

provider:
  name: aws
  runtime: nodejs12.x
  lambdaHashingVersion: 20201221
  httpApi:
    authorizers: test

functions:
  hello:
    handler: index.handler
```

which results in
```

  Serverless Error ---------------------------------------

  Configuration error at 'provider.httpApi.authorizers': should be object

```

with `configValidationMode` not elevated to `error`, it does produce the following warning:

```
Serverless: Configuration warning at 'provider.httpApi.authorizers': should be object
Serverless:
Serverless: Learn more about configuration validation here: http://slss.io/configuration-validation
```

Reply:
@BryanCrotaz Hello, did you see my previous comment?